connectors/spark/src/main/scala/io/unitycatalog/spark/UCSingleCatalog.scala

```diff
@@ -276,7 +276,7 @@ private class UCProxy(
         properties = t.getProperties.asScala.toMap ++ extraSerdeProps
       ),
       schema = StructType(fields),
-      provider = Some(t.getDataSourceFormat.getValue),
+      provider = Some(t.getDataSourceFormat.getValue.toLowerCase()),
```

Comment:
If we don't lower case here,  then we will encounter another problem: https://github.com/unitycatalog/unitycatalog/issues/1121#issuecomment-3444232124